### PR TITLE
Fix ESLint warnings

### PR DESCRIPTION
**Pull request by Frenzzy**
- Update ESLint version to fix `Unexpected space before function parentheses`
- Add brackets around `await` to it `Missing space before keyword "await"`
- Remove second blank lines to fix `Too many blank lines at the end of file. Max of 1 allowed`

t

### CODE REVIEW

**Comment by koistya** [comment]
It's better to use async methods inside Node.js app.

**Comment by Frenzzy** [comment]
Ok, PR updated!

File: tools/build.js [hunk 1]
```diff
@@ -20,4 +20,3 @@ async function build() {
 }
 
 export default build;
-
```
Comment (CombatCode) [inline tools/build.js]: http://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file

Comment (CombatCode) [inline tools/build.js]: Also, I'm reffering to project settings https://github.com/kriasoft/react-starter-kit/blob/master/.editorconfig#L14

Comment (Frenzzy) [inline tools/build.js]: This file has two newlines at the end instead of one newline, see airbnb/javascript/pull/578

**Pull request merged**